.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "src/coreclr"]
 	path = src/coreclr
-	url = https://github.com/dotnet/coreclr
+	url = https://github.com/dagood/coreclr
```

Comment:
Note a huge deal but I generally perfer to not switch to forks but instead just use a branch in the main repo.